libs/features/spectrum2-styles/s2-styles.css

```diff
@@ -0,0 +1,931 @@
+/* stylelint-disable selector-class-pattern */
```

Comment:
I would suggest to put the output of the file directly into `libs/deps/spectrum2-styles.css`, but for the poc here is completely fine 👍